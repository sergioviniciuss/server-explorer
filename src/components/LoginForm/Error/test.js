import React from "react";
import { render, screen } from "@testing-library/react";
import { Error } from ".";

describe("Error", () => {
  it("renders Error component with error message", () => {
    const { container } = render(<Error error="An error occurred..." />);

    expect(screen.getByText(/An error occurred.../i)).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
