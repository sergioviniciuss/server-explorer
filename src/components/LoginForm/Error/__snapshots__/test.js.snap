// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders Error component with error message 1`] = `
<span
  class="error"
>
  An error occurred...
</span>
`;
